schema/composition/animation.json

```diff
@@ -10,6 +10,12 @@
         {
             "type": "object",
             "properties": {
+                "ver": {
+                    "title": "Specification Version",
+                    "description": "Specification version this Lottie is targeting. This is a 6 digit number with version components encoded as `MMmmpp`, with `MM` being major version, `mm` being minor and `pp` being patch.",
+                    "type": "integer",
+                    "minimum": 10000
```

Comment:
We should probably specify explicit version numbers, someone publishing a lottie with version 2.0.0 `ver: 20000` would not be valid as there's no associated schema.

Reply:
I think the key question here is if lotties created in the future with a new version should be considered invalid for older schema?

If a future lottie otherwise passes an older schema, I think that should still be good enough to be considered valid.
 
I am concerned about extra work to keep schemas updated in all the tooling, particularly for smaller revisions and/or platforms that have a more involved release process, such as wear.